src/OpenTK.Mathematics/Vector/Vector2i.cs

```diff
@@ -0,0 +1,544 @@
+//
+//  Vector3i.cs
```

Comment:
Minor, but why are we repeating the file name in the comments? That seems really pointless.

Reply:
Good catch, that should be Vector2i. As a big picture discussion point, not a clue. Jax is the one to talk to about that.